<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissHistoryQuery</class>
 <widget class="QWidget" name="MissHistoryQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MissHistoryQuery</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblHistory">
     <property name="text">
      <string>历史查询:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSerial">
     <property name="text">
      <string>序列号:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtValue"/>
   </item>
   <item>
    <widget class="QComboBox" name="cboxOperator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateValue">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnQuery">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuery</sender>
   <signal>clicked()</signal>
   <receiver>MissHistoryQuery</receiver>
   <slot>s_btnQuery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtValue</sender>
   <signal>returnPressed()</signal>
   <receiver>btnQuery</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_btnQuery()</slot>
 </slots>
</ui>
